quipucords/api/scanjob/serializer.py

```diff
@@ -26,12 +28,14 @@ class ScanOptionsSerializer(NotEmptySerializer):
     """Serializer for the ScanOptions model."""
 
     max_concurrency = IntegerField(required=False, min_value=1, default=50)
+    disable_optional_products = CustomJSONField(required=False)
```

Comment:
I believe we need a good amount of validation logic added for this field. Since this is just taking JSON right now any valid JSON could be stored to the API. We need to add code here to restrict the appropriate JSON to an object with specific keys and values of True/False.

Reply:
Is there an example of this/something close to this? I am not sure exactly how to go about it (new to JSON).